projects/kubernetes-sigs/image-builder/build/setup_packer_configs.sh

```diff
@@ -214,3 +214,17 @@ if [ "$IMAGE_FORMAT" = "ami" ]; then
     fi
 fi
 
+EKSA_ANSIBLE_VERBOSITY="${EKSA_ANSIBLE_VERBOSITY:-}"
+if [ -n "$EKSA_ANSIBLE_VERBOSITY" ]; then
+    PACKER_FILE_NAME="packer.json"
```

Comment:
does this cover raw/nutanix as well?

Reply:
Yeah everything except OVA is packer.json